client/src/app/account/components/dialogue/dialogue.component.ts

```diff
@@ -122,4 +145,32 @@ export class DialogueComponent implements OnInit {
   getSymbol(currency: string): string {
     return getCurrencySymbol(currency, 'wide');
   }
+
+  setValues() {
+    if (this.account) {
+      this.addAccountForm.get('title')!.setValue(this.account.title);
+      this.addAccountForm
+        .get('description')!
+        .setValue(this.account.description);
+      this.addAccountForm.get('currency')!.setValue(this.account.currency);
+    }
+
+    let userId = localStorage.getItem('userId');
```

Comment:
Not critical, but query `localStorage` directly from `component` is pretty bad practice. I suggest create some service with `setUserId` and `getUserId` methods (or add them to some existing service), and inject it to component. Otherwise component takes too many responsibilities and it makes it difficult to refactor.

Reply:
Ok, got it. I knew it was bad practice but in order to advance and finish tasks, I just continued doing the project.